README.md

```
@@ -11,7 +11,7 @@ The config library is a significant extension to the args package and enables
 typed options, environment variables and configuration files as input, and
 better error reporting.
 
-[Config README](README_config.md)
+[Config README](#Config)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
Expected: #config; Actual: #Config

(MD051, link-fragments)